### PR TITLE
I18n sync partial up out 11 28

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: apps/i18n/craft/vi_vn.json [hunk 1]
```diff
@@ -1,34 +1,34 @@
 {
   "absolute": "giá trị tuyệt đối",
-  "agentGenericFailureMessage": "Keep coding! Remember to use The Agent to help.",
-  "agentTooFewBlocksFailureMessage": "Try adding just one block at a time to get The Agent to the right spot.",
+  "agentGenericFailureMessage": "Hãy tiếp tục lập trình! Nhớ hãy sử dụng Đặc vụ để giúp đỡ.",
+  "agentTooFewBlocksFailureMessage": "Hãy thử thêm chỉ 1 khối lệnh vào thời điểm Đặc vụ tới được đúng chỗ.",
   "at": "vào",
-  "agentDiamondPathCongrats": "Congratulations, you found the diamond path! You have collected {count, plural, one {1 diamond} other {# diamonds}} so far!",
+  "agentDiamondPathCongrats": "Xin chúc mừng, bạn tìm thấy đường dẫn kim cương! Đến thời điểm hiện tại bạn đã được thu thập {count, plural,one {1 kim cương}other {# kim cương}}!",
```
Comment (tanyaparker) [inline apps/i18n/craft/vi_vn.json:6]: Does the spacing matter?

File: apps/i18n/maze/vi_vn.json [hunk 2]
```diff
@@ -9,13 +9,13 @@
   "collectorCollectedEverything": "Xin chúc mừng! Bạn thu thập tất cả {count} món đồ",
   "collectorCollectedNothing": "Hãy tiếp tục mã hóa! Bạn cần phải thu thập càng nhiều của các món đồ có thể.",
   "collectorCollectedSome": "Tốt lắm! Bạn đã thu thập {count} món đồ!",
-  "collectorCollectedNotEnough": "Keep coding! You need to collect at least {goal, plural, one {1 item} other {# items}}.",
-  "collectorCollectedTooMany": "That space doesn't have enough items for me to collect.",
+  "collectorCollectedNotEnough": "Hãy tiếp tục lập trình! Bạn cần thu thập ít nhất {goal, plural,one {1 đồ vật}other {đồ vật}}",
```
Comment (tanyaparker) [inline apps/i18n/maze/vi_vn.json:12]: Spacing here too

File: pegasus/cache/i18n/vi-VN.yml [hunk 3]
```diff
@@ -2121,15 +2120,3 @@
     Classroom” button on the “Manage students” tab:'
   google_sign_in_instruct: 'From there, they should choose the “Sign in with Google
     Account” option:'
-  og_title_soon: The Hour of Code is coming
```
Comment (tanyaparker) [inline pegasus/cache/i18n/vi-VN.yml:2124]: Seems weird these strings were removed.

Comment (Hamms) [reply pegasus/cache/i18n/vi-VN.yml:2124]: looks like they were just added yesterday and my sync didn't include that change

File: apps/i18n/craft/vi_vn.json (hunk 1 above)
Comment (Hamms) [reply apps/i18n/craft/vi_vn.json:6]: tested manually, looks like it does not!

File: pegasus/cache/i18n/vi-VN.yml (hunk 3 above)
Comment (Hamms) [reply pegasus/cache/i18n/vi-VN.yml:2124]: However it also looks like the language is appropriately falling back to english, so I'm not worried about them missing

Comment (tanyaparker) [reply pegasus/cache/i18n/vi-VN.yml:2124]: I didn't know `I18n.t` falls back! Sweet. I'll just add to list of stuff to test in bug bash, just to be sure.

**Pull request merged**